app/utils/contentful.ts

```diff
@@ -21,7 +21,7 @@ export async function getEntryById(id: string) {
 export async function getPage() {
   const entry = await client.getEntry(LANDING_PAGE_ID, {
     include: 2,
-    locale: "es-US",
+    locale: "en-US",
```

Comment:
I had hardcode this value to test that translations work. Reverting for now. We need to make this dynamic in the future.

Reply:
Created ticket [LUS-105 Dynamic Locale Switching](https://linear.app/ofashandfire/issue/LUS-105/dynamic-locale-switching-en-es)